charm/config.yaml

```diff
@@ -32,6 +32,17 @@ options:
     github_webhook_secret:
         type: string
         description: GitHub webhook secret
+    poller_github_auth_token:
+        type: string
+        description: GitHub auth token to be used exclusively by the poller
+    poller_request_builds:
+        type: string
+        description: set to 'true' in other to allow poller to request builds
```

Comment:
I think that should be "in order", not "in other".

Reply:
typo, fixed. Thanks